src/Collection.php

```diff
@@ -552,4 +552,14 @@ public function merge($items)
 
         return $col;
     }
+
+    public function first()
+    {
+        return reset($this->items);
```

Comment:
@jakubgiminski @danielgsims Hey you can use `reset()` + `key()` in order to achieve desired behaviour. If `key()` returns  `NULL` that means there is no element in the array.